index.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
 
     app.import('vendor/node-webkit/shim.js', { prepend: true });
     app.import({ development: 'vendor/node-webkit/reload.js' });
+    app.import({ development: '../../node_modules/ember-cli/node_modules/testem/node_modules/socket.io/node_modules/socket.io-client/socket.io.js' });
```

Comment:
I would remove this import and just bring `socket.io` in via the `contentFor` hook. This will inject the script into `tests/index.html`:

```js
// ./index.js

contentFor: function(type) {
  // TODO Make testem server URL configurable
  var testemServer = 'http://localhost:7357';
  if (type === 'test-body' && process.env.EMBER_ENV === 'test') {
    return '<script src="' + testemServer + '/socket.io/socket.io.js"></script>';
  }
}
```